### PR TITLE
ocamlPackages.magic-mime: 1.0.0 -> 1.1.3

### PR DESCRIPTION
**Pull request by superherointj**
ocamlPackages.magic-mime: 1.0.0 -> 1.1.3

### CODE REVIEW

File: pkgs/development/ocaml-modules/magic-mime/default.nix [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
+  version = "1.1.3";
 
-stdenv.mkDerivation {
-  pname = "ocaml-magic-mime";
-  inherit version;
-
-  src = fetchzip {
-    url = "https://github.com/mirage/ocaml-magic-mime/archive/v${version}.tar.gz";
-    sha256 = "058d83hmxd5mjccxdm3ydchmhk2lca5jdg82jg0klsigmf4ida6v";
+  src = fetchFromGitHub {
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/magic-mime/default.nix:7]: Please use `fetchurl` with that URL: https://github.com/mirage/ocaml-magic-mime/releases/download/v1.1.3/magic-mime-v1.1.3.tbz

File: pkgs/development/ocaml-modules/magic-mime/default.nix [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
+    rev = "v${version}";
+    sha256 = "sha256-izoiJ3wXTA4TgZR2jEwtDh3qkzE5vyoJqQx8HbtLdjI=";
   };
 
-  nativeBuildInputs = [ ocaml findlib ocamlbuild ];
-  buildInputs = [ findlib ];
-  configurePlatforms = [];
-
-  createFindlibDestdir = true;
+  minimumOCamlVersion = "4.03";
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/magic-mime/default.nix:14]: Maybe you can use the brand new `minimalOCamlVersion`.

File: pkgs/development/ocaml-modules/magic-mime/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/development/ocaml-modules/magic-mime/default.nix]: Done. I am correcting this in every PR but you were faster than my queue. :-)

File: pkgs/development/ocaml-modules/magic-mime/default.nix (hunk 2 above)
Comment (superherointj) [reply pkgs/development/ocaml-modules/magic-mime/default.nix]: Sure. From now on will use `minimalOCamlVersion`.

**Pull request merged**